tools/online_editor/src/index.ts

```diff
@@ -96,8 +114,41 @@ function create_open_menu(editor: EditorWidget): Menu {
             editor.project_from_url(url);
         },
     });
+    commands.addCommand("slint:add_file", {
+        label: "Add a file to your Slint project",
+        iconClass: "fa-regular fa-file",
+        mnemonic: 1,
+        execute: () => {
+            let name = prompt("Please enter the file name");
+            if (name == null) {
+                return;
+            }
+            if (!name.endsWith(".slint")) {
+                name = "${name}.slint";
+            }
+            editor.add_empty_file_to_project(name);
+        },
+    });
+    commands.addCommand("slint:store_github_token", {
```

Comment:
seems strange. shouldn't that happen when saving to a gist? (as a first step)
This commands clutters the interface